dist/inputs/ObjectSearchCreate.d.ts

```diff
@@ -6,6 +6,7 @@ declare class ObjectSearchCreate extends Component<IInputProps> {
     private search;
     private readonly injected;
     private readonly fieldConfig;
+    private readonly selectProps;
```

Comment:
note: When the build command will run before deploying, the buttonProps will be added too:

![screen shot 2019-01-28 at 10 07 56 am](https://user-images.githubusercontent.com/1119991/51845010-a8b78c80-22e4-11e9-9983-3ec13825d826.png)